scripts/schema/cleaner.py

```diff
@@ -244,10 +251,7 @@ def single_line_short_override_description(schema_or_field, strict=True):
             continue
         error = single_line_short_check(field['short_override'], field['full'])
         if error:
-            if strict:
-                raise ValueError(error)
-            else:
-                ecs_helpers.strict_warning(error)
+            strict_warning_handler(error, strict)
 
 
 def check_example_value(field, strict=True):
```

Comment:
Maybe `check_example_value` should verify the example are valid per the `pattern`.

Reply:
Good idea! Added in https://github.com/elastic/ecs/pull/1834/commits/14bf0d151f5c9ff1c340b7fbde93bd4e693169a9

Reply:
Good idea! Added in https://github.com/elastic/ecs/pull/1834/commits/14bf0d151f5c9ff1c340b7fbde93bd4e693169a9